cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -116,6 +117,7 @@ data NodeConfiguration
 
        , ncMaybeMempoolCapacityOverride :: !(Maybe MempoolCapacityBytesOverride)
 
+       , ncLedgerDBBackend :: !BackingStoreSelectorFlag
```

Comment:
```suggestion
       , ncLedgerDBBackend :: !BackingStoreSelectorFlag
       
```
And maybe a comment to describe this section of the configuration? Something like "UTxO HD configs"

Reply:
Fixed, thanks!